doc/man/scons.xml

```diff
@@ -4244,6 +4244,7 @@ to serve as the test can be supplied in
 if not supplied,
 the default checks the ability to link against the specified
 <parameter>library</parameter>.
+<parameter>extra_libs</parameter> can be used to add additional libraries to link against.
```

Comment:
@mwichmann - what's the blurb to indicate this is added in NEXT_RELEASE?

Reply:
Since the xml files are not processed by Sphinx, it needs the hand-crafted version:
```xml
<para>
<emphasis>Changed in version NEXT_RELEASE:</emphasis>
Added the <parameter>extra_libs</parameter> parameter.
</para>
```

It's a little unclear whether an *existing* method getting a *new* parameter is considered "added" or "changed".  I'm tending to follow what CPython does, which is to consider that "changed" (though I admit that's not been consistent).

The corresponding markup in source code (aka the docstring) is:
```py
.. versionchanged:: NEXT_RELEASE
    Added the  *extra_libs* parameter.
```